src/lib.rs

```diff
@@ -280,8 +280,8 @@ mod test {
     #[test]
     fn test_encrypt_decrypt() {
         let (key, _) = derive_key("krabbetje", None);
-        let plain_text = "Hello, world!".repeat(1);
```

Comment:
This would just convert the `&'static str` to a `String` right? Why should we not just use a `&'static str`?

Reply:
I'm not sure if I get the question right, but I guess that's exactly what I did. Removing `.repeat(1)` makes the variable `plain_text` a `&'static str` instead of a `String` which it used to be before. Therefore, we can also omit the `&` a few lines later.